### PR TITLE
[ibexa/experience,commerce] Added configuration for Rate Limiter

### PR DESCRIPTION
**Pull request by Nattfarinn**
Required by: https://github.com/ibexa/corporate-account/pull/106

### CODE REVIEW

**Review by alongosz — APPROVED** [review]

File: ibexa/commerce/4.3/config/packages/ibexa.yaml [hunk 1]
```diff
@@ -118,3 +118,9 @@ framework:
     esi: true
     fragments: true
     http_method_override: true
+
+    rate_limiter:
+        corporate_account_application:
+            policy: 'fixed_window'
+            limit: 1
```
Comment (konradoboza) [inline ibexa/commerce/4.3/config/packages/ibexa.yaml:125]: Shouldn't we increase limit per 5 minutes period? `1` seems to be a little low to me.

Comment (Nattfarinn) [reply ibexa/commerce/4.3/config/packages/ibexa.yaml:125]: This is limit per unique user (IP). I guess multiple submissions from the very same user applying for multiple different companies will be a very rare case.

Comment (konradoboza) [reply ibexa/commerce/4.3/config/packages/ibexa.yaml:125]: Makes sense, thanks for the clarification.

**Review by webhdx — APPROVED** [review]

**Review by ciastektk — APPROVED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by alongosz — APPROVED** [review]

**Review by Steveb-p — APPROVED** [review]
A bit unfortunate that developers will need to adjust both Symfony and our environment variable (or at least one of them, because they're different) if they're going to be using one common lock implementation, but it's the best that we could think of I guess...

**Review by mnocon — APPROVED** [review]

**Pull request merged**